Tools/TestWebKitAPI/Tests/mac/WKWebViewMacEditingTests.mm

```diff
@@ -219,6 +219,70 @@ - (SlowTextInputContext *)_web_superInputContext
     NSString *hasSpellingMarker = [webView stringByEvaluatingJavaScript:@"internals.hasSpellingMarker(6, 9) ? 'true' : 'false'"];
     EXPECT_STREQ("false", hasSpellingMarker.UTF8String);
 }
+
+
+@interface SetMarkedTextWithNoAttributedStringTestCandidate : NSTextCheckingResult
+@end
+
+@implementation SetMarkedTextWithNoAttributedStringTestCandidate {
+    NSString *_string;
```

Comment:
I know this is testing code, but it's good habit to not introduce new raw pointer usage.

Reply:
lol all of this is just copy-pasted from another test -- will fix though